providers/outreach/metadata.go

```diff
@@ -7,6 +7,11 @@ import (
 	"github.com/amp-labs/connectors/common"
 )
 
+var (
+	metadataPageSize      string = "1"          //nolint:gochecknoglobals
+	metadataPageSizeQuery string = "page[size]" //nolint: gochecknoglobals
```

Comment:
Just making sure, is `size` supposed to be substituted or does this actually look like this?

Reply:
Looks like that. No substitution.